src/lib/containers/Facets.js

```diff
@@ -14,28 +22,85 @@ const DESELECT_ALL = "deselect all"
  */
 class CheckboxGroup extends React.Component {
 
+    constructor() {
+        super()
+        this.handleHover = this.handleHover.bind(this)
+        this.handleHoverExit = this.handleHoverExit.bind(this)
+    }
+
+    handleHover(event) {
+        if (event.target.tagName === "SPAN") {
+            event.target.className += " SRC-primary-background-color"
+        }
+    }
+    
+    handleHoverExit(event) {
+        if (event.target.tagName === "SPAN") {
+            let className = event.target.className
+            if (className.indexOf(" SRC-primary-background-color") !== -1 ) {
+                className = className.substring(0, className.indexOf(" SRC-primary-background-color"))
```

Comment:
assumption here that the background color class is the last one listed (otherwise all other css classes will be lost).